discovery_engine_core/core/src/storage/sqlite/dart_migrations.rs

```diff
@@ -392,6 +410,38 @@ mod tests {
             data.excluded_sources.into_iter().collect()
         );
 
+        let weight = storage
+            .source_reaction()
+            .fetch_source_reaction("source1")
+            .await
+            .unwrap();
+        assert_eq!(weight, Some(true));
+        // assert_eq!(weight, 1);
```

Comment:
:thought_balloon: actually `fetch_source_reaction` could return the weight, and then it would be more useful here... (but not essential for this pr)

Reply:
yes that was planed, it was just written in parallel with another PR by someone else so I originally planed to do so in a follow up PR. But I can now do it now. :+1: